haproxy/reload-haproxy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+
+exec killall -SIGUSR2 haproxy-systemd-wrapper
```

Comment:
Is this the right signal? I think it would be [SIGHUP](https://docs.docker.com/samples/library/haproxy/#reloading-config) instead.

Reply:
if you look at [haproxy-systemd-wrapper](https://github.com/haproxy/haproxy/blob/25f067ccec52f53b0248a05caceb7841a3cb99df/src/haproxy-systemd-wrapper.c#L288), you see that both do the same. to me `SIGHUP` has more meaning of "shutting down", than "reloading configuration". (yeah, i agree that many programs misuse it for reloading configuration)

Reply:
Wow, true! :)